.github/workflows/main.yaml

```diff
@@ -0,0 +1,12 @@
+on: pull_request

```

Comment:
This changed since I originally wrote this, but I think we want to explicitly list types so we get `edit` as a type here as well: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#pull-request-event-pull_request